packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -81,8 +81,9 @@ export function generateRuntimeOptions(
         flushMode: [undefined],
         compressionOptions: [{ minimumBatchSizeInBytes: 500, compressionAlgorithm: CompressionAlgorithms.lz4 }],
         maxBatchSizeInBytes: [undefined],
+        // Compressed payloads over 1MB will be split into chunked ops of this size
+        chunkSizeInBytes: [614400],
```

Comment:
do you want to use here some combos, like chunkSizeInBytes: [1024, 614400], ?
Just to make sure system works with small and big numbers.